docs/docs/lib/csharp.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: C# SDK
+description: GrowthBook SDK for C#
+sidebar_label: C#
```

Comment:
To make this show up in the sidebar, you also have to edit `sidebars.js` and add this to the array on line 100.

In addition, on `docs/lib/index.mdx`, there is a link to the C# github repo.  We should probably change that to point to this new page instead.